install/index.md

```diff
@@ -274,6 +276,18 @@ RAPIDS pip packages are available for CUDA 11 and CUDA 12 on the NVIDIA Python P
 <i class="fas fa-info-circle"></i> **glibc version:** x86_64 wheels require glibc >= 2.17. <br/>
 <i class="fas fa-info-circle"></i> **glibc version:** ARM architecture (aarch64) wheels require glibc >= 2.32 (only ARM Server Base System Architecture is supported).
 
+<br/>
+<div id="sdkm"></div>
+
+### **SDK Manager (Ubuntu Only)**
+[NVIDIA's SDK Manager](https://developer.nvidia.com/sdk-manager) gives a users a Graphical User Interface (GUI) option to install RAPIDS.  It also attempts to fix any environment issues before installing RAPIDS or updating RAPIDS, making it ideal for new Linux users.
+1. Download [SDK Manager's Ubuntu version from their website](https://developer.nvidia.com/sdk-manager){: target="_blank"} (requires sign up or login to NVIDIA's Developer community).  Do not install yet.  It is assumed that your home directory's `Downloads` folder is where the `.deb` file will be stored.  If not, please move `sdkmanager_[version]-[build#]_amd64.deb` file to your current Download folder.
```

Comment:
Why does it matter if this is in your `Downloads` folder? We should not have to dictate where users download files, so long as they are accessible.

Reply:
This is very true.  However, I was attempting to simplify the instructions and levelset where most Windows users would have their downloads files.  Thoughts on a different easy way to allow for these users to find the `.deb` file and then use it for the `sudo apt install`?  Remember that some of these users aren't adept with CLI commands from either Windows or Ubuntu.

Reply:
I think the specificity here is actually harmful. I would keep it short. "Download the `.deb` file and navigate to that directory in the WSL terminal."

Reply:
changed in the latest commit